app/views/shared/_user_standing.html.erb

```diff
@@ -0,0 +1,131 @@
+<div class="bg-gray-50 md:mt-16 h-full">
+  <div class="py-8 md:py-12 border-b border-gray-200 bg-white">
+    <div class="max-w-5xl mx-auto px-5 flex flex-col md:flex-row items-start justify-between gap-1">
+      <div class="flex gap-1 items-center">
+        <div class="w-15 h-15 border-gray-200 rounded-full overflow-hidden shrink-0 m-1">
+          <% if @presenter.user.avatar.attached? %>
+            <%= image_tag @presenter.user.avatar_url(variant: :thumb), alt: 'Avatar', class: "w-full h-full" %>
+          <% else %>
+            <div data-re-component="Avatar" data-re-json="<%= { name: @presenter.user.name }.to_json %>"></div>
+          <% end %>
+        </div>
+        <div>
+          <p class="text-base font-semibold"><%= @presenter.user.name %></p>
+          <p class="text-sm text-gray-500"><%= @presenter.user.title %></p>
+        </div>
+      </div>
+      <% if @presenter.standing_enabled? %>
+        <div class="flex flex-row-reverse md:flex-row items-center justify-start md:justify-start gap-2">
+          <div class="text-left rtl:text-right rtl:md:text-left md:text-right">
+            <p class="font-semibold text-sm" style="color: <%= @presenter.current_standing.color %>;"><%= @presenter.current_standing.name %></p>
+            <div class="flex gap-1 items-center justify-start md:justify-end">
+              <p class="text-sm text-gray-500"><%= t(".standing") %></p>
+              <% if @presenter.standing_help? %>
+                <p data-re-component="HelpIcon" data-re-json='<%= @presenter.help_props.to_json %>'></p>
```

Comment:
Yes, it is an option to show the description in the current standing section, but It could to lead to confusion for user with reason of the latest standing log which is right below it, without a proper title (Description). 😁🤔.

You have also removed the link (Read more) which takes to documentation page why ?

If we decide to show the description of the standing in the current standing then I would like to add what standing is and read more in the help icon dropdown.

@irajsuhail what are your thoughts ?

Reply:
> You have also removed the link (Read more) which takes to documentation page why ?

@yash-learner We keep LMS doc links only on the admin side. The user-facing side of the LMS is white-labelled, so we've kept Pupilfirst branded things out of it.

When additional help / documentation is genuinely required for a user-facing feature, we've kept it on the user-side of the LMS. For example: https://www.pupilfirst.school/help/markdown_editor (which is linked from Markdown editors)

> If we decide to show the description of the standing in the current standing then I would like to add what standing is and read more in the help icon dropdown.

Given that different schools will have different standings, shouldn't we allow the description of the standings to act as the documentation of what they and what they mean within the context of that school? 🤔

Reply:
Yeah, Sure. Let's show the description of standings in the standing page.

Reply:
> We keep LMS doc links only on the admin side. The user-facing side of the LMS is white-labelled, so we've kept Pupilfirst branded things out of it.

Thank you for the info. 🙌

----

Updated the code to show the description in current standing card.

---

> Given that different schools will have different standings, shouldn't we allow the description of the standings to act as the documentation of what they and what they mean within the context of that school?

Yes